man/tpm2_policycommandcode.1.md

```diff
@@ -0,0 +1,69 @@
+% tpm2_policycommandcode(1) tpm2-tools | General Commands Manual
+%
+% JANUARY 2018
+
+# NAME
+
+**tpm2_policycommandcode**(1) - Restrict TPM object authorization to specific TPM commands or operations.
+
+# SYNOPSIS
+
+**tpm2_policycommandcode** [*OPTIONS*]
+
+# DESCRIPTION
+
+**tpm2_policycommandcode**(1) Restricts TPM object authorization to specific TPM commands or operations. Useful when you want to allow only specific commands with the TPM object. Requires support for extended sessions with resource manager.
+
+# OPTIONS
+
+  * **-S**, **--session**=_SESSION\_FILE_:
+
+    A session file from **tpm2_startauthsession**(1)'s **-S** option.
+
+  * **-c**, **--command-code**=_TPM\_CC\_VALUE_:
+
+    A command code of the TPM operation the object auth should be restricted to.
+
+   * **-o**, **--policy-file**=_POLICY\_FILE_:
+
+    File to save the policy digest.
+[common options](common/options.md)
+
+[common tcti options](common/tcti.md)
+
+# EXAMPLES
+
+Start a *policy* session and extend it with a specific command like unseal. Attempts to perform other operations would fail.
+
+## Create an unseal-only policy
+```
+TPM_CC_UNSEAL=0x15E
+tpm2_startauthsession -S session.dat
+tpm2_policycommandcode -S session.dat -c $TPM_CC_USEAL -o policy.dat
+tpm2_flushcontext -S session.dat
+```
+## Create the object with unseal-only auth policy
+```
+tpm2_createprimary -a o -o prim.ctx
+tpm2_create -C prim.ctx -u sealkey.pub -r sealkey.priv -L policy.dat -I- <<< "SEALED-SECRET"
+```
+# Try unseal operation
+```
+tpm2_load -C prim.ctx -u sealkey.pub -r sealkey.priv -n sealkey.name -o sealkey.ctx
+tpm2_startauthsession -a -S session.dat
+tpm2_policycommandcode -S session.dat -o policy.dat -c $TPM_CC_UNSEAL
+tpm2_unseal -p session:session.dat -c sealkey.ctx
+tpm2_flushcontext -S session.dat
+```
+
```

Comment:
Why the newline space here but not the others. I like the whitespace separating the blocks, I would add a new line between the blocks above.

Reply:
Still open.